README.md

```diff
@@ -84,3 +91,4 @@ These are only the graphics part of an engine but mature enough and well develop
 |![magnum](https://github.com/redorav/public_source_engines/blob/master/logos/magnum_64px.png)| Magnum | https://github.com/mosra/magnum |
 |![vultaik](https://github.com/redorav/public_source_engines/blob/master/logos/vultaik_64px.png)| Vultaik | https://github.com/FaberSanZ/Vultaik |
 || Veldrid | https://github.com/mellinoe/veldrid |
+|![embree](logos/embree_64px.png)|Intel Embree |https://github.com/RenderKit/embree |
```

Comment:
I'm not quite sure about this one, this is a raytracing framework that isn't really used in realtime, it's not a renderer or anything like what's already on this list